homeassistant/components/sensor/darksky.py

```diff
@@ -127,7 +130,9 @@
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Setup the Dark Sky sensor."""
     # Validate the configuration
-    if None in (hass.config.latitude, hass.config.longitude):
+    latitude = config.get(CONF_LATITUDE, hass.config.latitude)
```

Comment:
This is weird. Now it's possible to use HASS latitude but use configured longitude? Please make it either.